### DISCUSSION TITLE
IBAN nella fattura elettronica formato PDF con API

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#146] [Q&A] [answered]
Salve,
come posso visualizzare l'IBAN nella fattura elettronica formato PDF creata tramite le API?
Ecco il mio json

{class IssuedDocument 
{  Id:   Entity: class Entity 
{  Id:   Code:   Name: ZZZ ZZZ  Type: Person  FirstName: ZZZ  LastName: ZZZ  ContactPerson:   VatNumber:   TaxCode: ZZZ  AddressStreet: ZZZ  AddressPostalCode: 10100  AddressCity: Torino  AddressProvince: TO  AddressExtra:   Country: Italia  Email: ZZZ  CertifiedEmail:   Phone:   Fax:   Notes:   DefaultVat:   DefaultPaymentTerms:   DefaultPaymentTermsType:   DefaultPaymentMethod:   BankName:   BankIban:   BankSwiftCode:   ShippingAddress:   EInvoice:   EiCode:   HasIntentDeclaration:   IntentDeclarationProtocolNumber:   IntentDeclarationProtocolDate:   CreatedAt:   UpdatedAt: }
  Type: Invoice  Number: 7  Numeration:   Date: 07/11/2022 00:00:00  Year: 2022  
  Currency: class Currency {  Id: EUR  Symbol:   ExchangeRate:   HtmlSymbol: }  
  Language: class Language {  Code: it  Name: }  
  Subject: ZZZ  VisibleSubject: ZZZ  RcCenter:   Notes:   Rivalsa:   Cassa:   AmountCassa:   CassaTaxable:   AmountCassaTaxable:   Cassa2:   AmountCassa2:   Cassa2Taxable:   AmountCassa2Taxable:   GlobalCassaTaxable:   AmountGlobalCassaTaxable:   WithholdingTax:   WithholdingTaxTaxable:   OtherWithholdingTax:   StampDuty:   
  PaymentMethod: class PaymentMethod {  Id:   Name: Bonifico bancario  Type: Standard  IsDefault:   DefaultPaymentAccount:   Details:   BankIban: ZZZ  BankName: UNICREDIT  BankBeneficiary: UNICREDIT  EiPaymentMethod: ZZZ}  
  UseSplitPayment:   UseGrossPrices:   
  EInvoice: True  
  EiData: class IssuedDocumentEiData {  VatKind: I  OriginalDocumentType: Ordine  OdNumber: 7  OdDate: 07/11/2022 00:00:00  Cig:   Cup:   PaymentMethod: MP05  BankName: UNICREDIT  BankIban: ZZZ  BankBeneficiary:   InvoiceNumber:   InvoiceDate: }  
  EiCassaType:   EiCassa2Type:   EiWithholdingTaxCausal:   EiOtherWithholdingTaxType:   EiOtherWithholdingTaxCausal:   ItemsList: System.Collections.Generic.List`1[It.FattureInCloud.Sdk.Model.IssuedDocumentItemsListItem]  PaymentsList: System.Collections.Generic.List`1[It.FattureInCloud.Sdk.Model.IssuedDocumentPaymentsListItem]  Template:   DeliveryNoteTemplate:   AccInvTemplate:   HMargins:   VMargins:   ShowPayments:   ShowPaymentMethod: True  ShowTotals:   ShowPaypalButton:   ShowNotificationButton:   ShowTspayButton:   DeliveryNote:   AccompanyingInvoice:   DnNumber:   DnDate:   DnAiPackagesNumber:   DnAiWeight:   DnAiCausal:   DnAiDestination:   DnAiTransporter:   DnAiNotes:   IsMarked:   AmountNet:   AmountVat:   AmountGross:   AmountDueDiscount:   AmountRivalsa:   AmountRivalsaTaxable:   AmountWithholdingTax:   AmountWithholdingTaxTaxable:   AmountOtherWithholdingTax:   AmountOtherWithholdingTaxTaxable:   AmountEnasarcoTaxable:   ExtraData:   SeenDate:   NextDueDate:   Url:   AttachmentUrl:   AttachmentToken:   EiRaw:   EiStatus: }}

**Comment by emajo** [comment, marked as answer]
Ciao @Mustino2022, per visualizzare la mail all'interno del pdf della fattura come primo step è necessario creare il cliente attraverso la apposita chiamata [Create Client](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/entities/clients) dove è necessario specificare il campo `email`, successivamente dovrai inserire il campo `entity.id` nella creazione della fattura popolandolo con l'id del cliente appena creato.

Grazie,
Emanuele Saccomandi.

**Reply by Mustino2022** [reply]
Ciao @emajo, la mia richiesta è relativa non all'email del cliente, ma a visualizzare l'IBAN nella fattura elettronica formato PDF
Grazie

**Reply by emajo** [reply]
Scusami, errore mio, per far vedere l'iban nel pdf dovrai prima creare un payment method con la [chiamata apposita](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/settings/payment_methods), oppure da web (è un operazione da compiere soo una volta), successivamente dovrai inserire il campo `payment_method.id` nella creazione della fattura popolandolo con l'id del metodo di pagamento appena creato.

**Reply by Mustino2022** [reply]
Ciao @emajo grazie per il supporto, funziona.
